akka-actor-typed/src/main/scala/akka/actor/typed/internal/adapter/ActorSystemAdapter.scala

```diff
@@ -25,15 +25,19 @@ import akka.typed.EventStream
 @InternalApi private[akka] class ActorSystemAdapter[-T](val untyped: a.ActorSystemImpl)
   extends ActorSystem[T] with ActorRef[T] with internal.ActorRefImpl[T] with ExtensionsImpl {
 
+  loadExtensions()
+
   import ActorRefAdapter.sendSystemMessage
 
   // Members declared in akka.actor.typed.ActorRef
   override def tell(msg: T): Unit = {
-    if (msg == null) throw new InvalidMessageException("[null] is not an allowed message")
+    if (msg == null) throw InvalidMessageException("[null] is not an allowed message")
```

Comment:
new should be quicker though, ignore warning?

Reply:
You think it'll make a difference? Not on any hot path, is a bug in user code to hit it

Reply:
Yeah hm, I don't mind either I guess ;)